challenges/selectionSort/selection-sort.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function selectionSort(intArr) {
+  let n = intArr.length;
+  for (let i = 0; i <= n-1; i++) {
+    let min = i;
+    for (let j = i + 1; j !== n; j++) {
+      if (intArr[j] < intArr[min]) min = j;
+    }
+    let temp = intArr[min];
+    intArr[min] = intArr[i];
+    intArr[i] = temp;
+  }
```

Comment:
Also, you are going to need to have a readme/blog for this individual sort which should have an explanation of what insert sort is, walking through the algorithm that you were provided, that would like:
- At first pass this is what happens (insert your visual)
- At second pass this is what happens (insert your visual)
....
- Finally at last pass this is what will happen: (insert your visual)